docs/walkthroughs/developing-apbs-locally.adoc

```diff
@@ -46,11 +44,48 @@ apb init
 
 NOTE: Best practice is to name the APB ending with `-apb`.
 
-== Making your APB consumable as a mobile service
+== General APB considerations
+
+=== Scaling
+
+Use a link:https://kubernetes.io/docs/concepts/workloads/controllers/deployment/[Deployment] with a link:https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/[ReplicaSet] for containers that can scale horizontally. This will allow users of the APB to scale the service if needed.
```

Comment:
Do you really need to know about the ReplicaSet? As far as i know it's sufficient to use the `replicas` property of a `Deployment` and your replicas will be managed automatically: https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#replicas

Reply:
Fair point. I'll reword it to mention just the `replicas` property as thats sufficient to mention here.

Reply:
done